vpn-network/vpn-network-impl/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
 
     implementation Kotlin.stdlib.jdk7
     implementation "com.squareup.logcat:logcat:_"
-    implementation "com.github.duckduckgo:netguard:1.0.0"
+    implementation "com.duckduckgo.netguard:netguard-android:1.1.0"
```

Comment:
Why not set this to _ and let the version handled by versions.properties?

Reply:
oh, absolutely, forgot about that :D Doing it now